ethereum/core/src/test/java/org/hyperledger/besu/ethereum/privacy/OnchainPrivacyControllerTest.java

```diff
@@ -0,0 +1,155 @@
+/*
+ * Copyright ConsenSys AG.
```

Comment:
nit: new files should have Copyright contributors to Hyperledger Besu

Reply:
Done